guides.Rmd

```diff
@@ -196,136 +317,260 @@ Note the use of `%o%` to quickly generate the multiplication table, and that the
 
-*   `date_breaks` and `date_minor_breaks()` allows you to position breaks by
-    date units (years, months, weeks, days, hours, minutes, and seconds). 
-    For example, `date_breaks = "2 weeks"` will place a major 
-    tick mark every two weeks.
+`r columns(3)`
+```{r}
+base <- ggplot(mpg, aes(displ, hwy, colour = cyl)) + geom_point()
 
-*   `date_labels` controls the display of the labels
-    using the same formatting strings as in `strptime()` and `format()`:
+base 
+base + scale_colour_continuous(guide = guide_coloursteps())
+base + guides(colour = guide_coloursteps())
+```
 
-    | String        | Meaning
-    |---------------|-----------------------------------------
-    | `%S`          | second (00-59)
-    | `%M`          | minute (00-59)
-    | `%l`          | hour, in 12-hour clock (1-12)
-    | `%I`          | hour, in 12-hour clock (01-12)
-    | `%p`          | am/pm
-    | `%H`          | hour, in 24-hour clock (00-23)
-    | `%a`          | day of week, abbreviated (Mon-Sun)
-    | `%A`          | day of week, full (Monday-Sunday)
-    | `%e`          | day of month (1-31)
-    | `%d`          | day of month (01-31)
-    | `%m`          | month, numeric (01-12)
-    | `%b`          | month, abbreviated (Jan-Dec)
-    | `%B`          | month, full (January-December)
-    | `%y`          | year, without century (00-99)
-    | `%Y`          | year, with century (0000-9999)
+However, scale guides are more complex than scale names: where the `name` argument (and hence the `labs()` helper function) takes text as input, the `guide` argument (and `guides()` helper) expect you to create a guide object using a **guide function** such as `guide_colourbar()` and `guide_legend()`. These functions offer the user additional control over the fine details of the legend. Legend guides can be used for any aesthetic (discrete or continuous) while the colour bar guide can only be used with continuous colour scales. 
 
-    For example, if you wanted to display dates like 14/10/1979, you would use 
-    the string `"%d/%m/%Y"`.
 
-The code below illustrates some of these parameters.
+The guide functions have numerous examples in their documentation help pages that illustrate all of their arguments. Most of the arguments to the guide function control the fine level details of the text colour, size, font etc. You'll learn about those in the themes chapter. Here I'll focus on the most important arguments.
```

Comment:
Generally, you won't need to use these arguments unless you want one guide to look different to the others.